src/utils/helper.rs

```diff
@@ -42,6 +44,37 @@ pub mod Util {
     pub const SGINFO: &str = "/usr/bin/sginfo";
     pub const JBOD_EXPORTER: &str = "/usr/bin/prometheus-jbod-exporter";
 
+    /// Returns an enum with true or false if a directory is empty
+    ///
+    /// This function verify if the directory is empty.
+    ///
+    /// # Arguments
+    ///
+    /// * `path` - a string reference
+    pub fn is_folder_empty(path: &str) -> io::Result<bool> {
+        Ok(fs::read_dir(path)?.next().is_none())
+    }
+
+    /// Verify if sysclass folder exists or otherwise exit and
+    /// provide software alternatives to jbod-rs.
+    pub fn verify_sysclass_folder(path: &str) {
+        if is_folder_empty(path).unwrap_or(false) {
+            println!(
+                "{} {} {}",
+                "==> ".red().bold(),
+                "jbod-rs not supported in this machine".bold(),
+                " <==".red().bold()
+            );
+            print!("{}", ":: ".bold().red());
+            println!("Use one of these alternatives: ");
+            println!(" - lsscsi");
+            println!(" - sas-lsi-tools");
+            println!(" - sas2ircu-status");
+            println!(" - sg3-utils");
```

Comment:
Not blocking merge: Didn't picked it first time but good practice would suggest using `STDERR` for reporting error so `eprintln!`.

It can be noted as an easy issue to review use of `println` and either use `eprintln` where appropriated.